python-ci/README.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Python + pip + ruff + pytest
+description: Set up a CI pipeline for a Python application with linting and testing
+tags: ["CI", "Python", "pip", "pytest", "ruff", "junit"]
+author: Buildkite
+---
+
+# CI for Python with pip, pytest and Ruff
+
+This template gives you a continuous integration (CI) pipeline that lints, tests, and builds a Python application.
+
+At a glance:
+
+- For [Python](https://www.python.org/) 3 applications
+- Uses [pip](https://pypi.org/project/pip/)
+- Requires [Docker](https://docs.docker.com/get-docker/)
+- Lints code with [Ruff](https://docs.astral.sh/ruff/)
+- Tests code using [pytest](https://docs.pytest.org/)
+
+## How it works
+
+This template:
+
+- Install Python dependencies with pip, caching the result.
+- Performs static analysis on the codebase with Ruff.
+- Runs pytest unit tests.
+- Automatically annotates the build based on junit test output
```

Comment:
Should we link to JUnit here?

Reply:
I was trying to find a decent link for junit output rather than junit itself, but struggled. I'd rather not link to Junit because it's not super relevant to the pipeline so I think I'll go without.